doc_src/faq.rst

```diff
@@ -174,15 +174,15 @@ add a statement to your :ref:`user initialization file <initialization>` (usuall
 
 How do I customize my syntax highlighting colors?
 -------------------------------------------------
-Use the web configuration tool, :ref:`fish_config <cmd-fish_config>`, or alter the `fish_color family of environment variables <index#variables-color>`__.
+Use the web configuration tool, :ref:`fish_config <cmd-fish_config>`, or alter the `fish_color family of environment variables <index.html#variables-color>`__.
```

Comment:
These links should lead to a *section* (via `:ref:\`something <variables-color>\``), not a .html file, so it can also work in other exports.

Reply:
Ah, much better. I actually forgot about other contexts for a second.